docs/typescript/schemas.md

```diff
@@ -95,7 +95,7 @@ The second generic param, `M`, is the model used with the schema. Mongoose uses
 
 The third generic param, `TInstanceMethods` is used to add types for instance methods defined in the schema.
 
-The 4th param, `TQueryHelpers`, is used to add types for [chainable query helpers](/docs/typescript/query-helpers.html).
+The 4th param, `TQueryHelpers`, is `used to add types for [chainable query helpers](query-helpers.html).
```

Comment:
why was this changed?

Reply:
this is the simplest relative link:
1. origin: `/docs/typescript/schemas.html`
2. relative link: `query-helpers.html`
3. becomes: `/docs/typescript/query-helpers.html`

OH accidentally fat-fingered a back tick here if that is what you are referring to